data/json/items/ranged/archery.json

```diff
@@ -331,15 +331,15 @@
     "effects": [ "IGNITE" ]
   },
   {
-    "id": "selfbow",
```

Comment:
This will create all sorts of errors on game start. You have to migrate existing "selfbow" items to "shortbow_makeshift" item, or leave changing the I'd altogether.

Reply:
Bizarre. I was sure I'd dealt with this already, and sure enough:

![image](https://user-images.githubusercontent.com/63127974/78435802-ac0a7e80-7633-11ea-8f17-4a15e434aa82.png)

I double checked, and the screenshotted version _is_ committed, yet your snippet (unlike Brian's) isn't showing up as 'outdated'. What am I missing?

Reply:
Oh, I think I see the problem: you were referring to the id name change of the selfbow - not, as I thought, to a phantom version where I'd maintained the same id. I did search all the JSON files for references to 'selfbow' and change them to the new id - it just happens that there were only two random NPCs that referenced it. That seemed strange at first, but it makes sense given my anecdotal experience of never encountering one in the wild.

Reply:
That's good, but more is required.  You must also add a migration entry to `data/json/items/migration.json` so the game knows how to replace existing `selfbow` items in on-going games.